rest/fetchPodcastEpisodeTranscript.ts

```diff
@@ -49,6 +54,7 @@ export const fetchPodcastEpisodeTranscript = async (
 
   transcriptProxyUrl.searchParams.set('type', targetTranscript.type);
   transcriptProxyUrl.searchParams.set('url', targetTranscript.url);
+  transcriptProxyUrl.searchParams.set('duration', duration.toString());
```

Comment:
This is typically how I convert to a string now:
```suggestion
  transcriptProxyUrl.searchParams.set('duration', `${duration}`);
```